c/httpserver.c

```diff
@@ -5821,7 +5821,9 @@ static int httpHandleTCP(STCBase *base,
         }
       }
 #endif // USE_ZOWE_TLS
-      ShortLivedHeap *slh = makeShortLivedHeap(READ_BUFFER_SIZE,100);
+      HttpServer *server = (HttpServer*) module->data;
+      unsigned int maxBlocks = server->config->httpRequestHeapMaxBlocks;
```

Comment:
This field should probably be initialized by this point in the `httpserver.c` code. The reason is, the `httpserver.c` might be used by non-zss applications which don't know about this filed and might not initialize it.